Zappy_IA_Report/DTC/Code/ml_agent.py

```
@@ -0,0 +1,192 @@
+
+
+def save_model(clf):
+    joblib.dump(clf, MODEL_FILE)
+
+
+def load_model():
+    return joblib.load(MODEL_FILE)
+
+
+def load_or_train_model():
+    global model, feature_names
```

Flake8 (7.2.0)

[error] `global feature_names` is unused: name is never assigned in scope

(F824)